src/ReservoirPriceOracle.sol

```diff
@@ -238,10 +237,10 @@ contract ReservoirPriceOracle is IPriceOracle, IReservoirPriceOracle, Owned(msg.
             lPayoutAmt = block.basefee * rewardGasAmount;
         }
 
-        if (lPayoutAmt <= address(this).balance) {
-            payable(aRecipient).transfer(lPayoutAmt);
+        // does not revert under any circumstance
+        assembly ("memory-safe") {
+            let result := call(gas(), aRecipient, lPayoutAmt, codesize(), 0x00, codesize(), 0x00)
```

Comment:
result is unused

Reply:
okay replaced it with a `pop()` to clear the stack.

It was either using `pop()` or to assign it. I guess popping the return value is a better solution